plugins/modules/net_tools/nmcli.py

```diff
@@ -95,6 +95,11 @@
             - Set metric level of ipv4 routes configured on interface.
         type: int
         version_added: 2.0.0
+    routing_rules4:
+        description:
+            - Is the same as in an 'ip route add' command, except always requires specifying a priority.
```

Comment:
```suggestion
            - Is the same as in an C(ip route add) command, except always requires specifying a priority.
```
Can you add an example using this option (and potentially also the others)? I think that would help understanding what this does.